### DISCUSSION TITLE
ZKSYNC Deployment Error

### DISCUSSION THREAD

**Discussion by PrincexMacbay** [Cyfrin/foundry-full-course-cu#2459] [Q&A]
image: https://github.com/user-attachments/assets/a9689435-cff8-4711-9c56-d2a74993f211
I am trying to deploy a smart contract using ZKSYNC with zkSync Sepolia Testnet. Immediately after signing the transaction it shows: 
``` You are using the latest version of the zksync plugin: v5.0.0-0
deploying contract "file-name" ```

But then in the deployment section, it says "contract failed to deploy"

**Comment by EngrPips** [comment]
Hello, are you working in the `default-workspace` and did you have all your contract files inside of the `contracts` folder

**Reply by PrincexMacbay** [reply]
No, I am not working in a default workspace and the files are
![Screenshot 2024-09-19 185559](https://github.com/user-attachments/assets/b50d3aca-2f40-40c5-ae33-8b7d799a6a96)
 in the contracts folder.

**Reply by EngrPips** [reply]
Did you remove the `zksync` plugin already?

**Reply by PrincexMacbay** [reply]
Yes I did

**Reply by EngrPips** [reply]
Well, `zksync` stuff isn't compulsory, so you can skip them if they are not working and work with sepolia.

**Reply by PrincexMacbay** [reply]
Okay EngrPips, Thank you very much.

**Reply by EngrPips** [reply]
Anytime, Friend.

**Comment by scullbeg24** [comment]
I had this issue. Possible answer is that if you have connected via wallet that you have initially not connected to correct zksync network. i removed and re-added zksync plugin, chose the  zksync sepolia testnet, recompiled, deployed, and it worked.

**Comment by EngrPips** [comment]
Thanks for the heads-up @scullbeg24
